<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportForm</class>
 <widget class="QDialog" name="ReportForm">
  <property name="windowTitle">
   <string>Select Report Contents</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the information to include in the report:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFileName">
     <property name="text">
      <string>File Name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkVersion">
     <property name="text">
      <string>LAS Version</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkPointFormat">
     <property name="text">
      <string>Point Format</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkNumPoints">
     <property name="text">
      <string>Number of Points</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBounds">
     <property name="text">
      <string>Bounds (Min/Max)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkXAxisBounds">
     <property name="text">
      <string>X-Axis Bounds</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkYAxisBounds">
     <property name="text">
      <string>Y-Axis Bounds</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
